manifests/init.pp

```diff
@@ -29,7 +29,7 @@
 define local_user (
   $state,
   $groups,
-  $password,
+  $password            = '*',
```

Comment:
I'm not sure if one is better than the other... I could go either way, do you have a preference?